File: src/vel_pkg/scripts/vel_node.py
#!/usr/bin/env python3
import rospy
from geometry_msgs.msg import Twist
def main():
    rospy.init_node("vel_node_py")
    pub_vel = rospy.Publisher("cmd_vel",Twist,queue_size=1)
    msg = Twist()
    msg.linear.x = 1
    msg.angular.z = 2
    rate = rospy.Rate(10)
    while not rospy.is_shutdown():
        pub_vel.publish(msg)
        rate.sleep()

if __name__ == '__main__':
    main()
    

File: src/vel_pkg/src/vel_node.cpp
#include<ros/ros.h>
#include<geometry_msgs/Twist.h>
int main(int argc, char *argv[])
{
    ros::init(argc,argv,"vel_node");
    ros::NodeHandle nh;
    ros::Publisher pub_vel = nh.advertise<geometry_msgs::Twist>("cmd_vel",1);
    geometry_msgs::Twist cmd_vel;
    cmd_vel.linear.x = 0.5;
    cmd_vel.angular.z = 1.0;
    ros::Rate rate(10);
    while (ros::ok())
    {
        pub_vel.publish(cmd_vel);
    }
    
    return 0;
}
